docs/csharp/language-reference/tokens/interpolated.md

```diff
@@ -32,7 +32,7 @@ The structure of an item with an interpolation expression is as follows:
 {<interpolationExpression>[,<alignment>][:<formatString>]}
 ```
 
-Elements in square brackets are optional. The following table describes each element:
+Elements in square brackets are optional. The following table describes each element:    
```

Comment:
I guess that was a typo to add unnecessary white spaces:
```suggestion
Elements in square brackets are optional. The following table describes each element:
```

Reply:
Good catch Petr. I deleted the extra spaces.